### ISSUE TITLE
[Android] Multiple controllers in Rayman Legends Definitive Edition is not working

### ISSUE THREAD

**Issue by H0nzik1** [yuzu-emu/yuzu#13078]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

android-256

### Description of Issue

I cannot add 2nd character ingame Rayman Legends Definitive Edition. While controls settings are set correctly and are working in other games it is not working for Rayman Legends Definitive Edition. 

### Expected Behavior

Able to add more controllers/heros ingame.

### Reproduction Steps

Just start game with multiple controllers and try to add more heros.

### Log File

[yuzu_log.txt](https://github.com/yuzu-emu/yuzu/files/14329402/yuzu_log.txt)

### System Configuration

CPU: Snapdragon 8 Gen 2
GPU/Driver: Adreno 740 / Mesa Turnip R17-V2
RAM: 8GB
OS: Android 14
Samsung S23

**Comment by EgoBizarro** [comment]
Yeah, this one is confusing. It's not yuzu's fault, though. You probably have controller applet disabled. You can only set the controllers at the start of the game when the applet pops up.

**Comment by H0nzik1** [comment]
> Yeah, this one is confusing. It's not yuzu's fault, though. You probably have controller applet disabled. You can only set the controllers at the start of the game when the applet pops up.

What is controller applet? I am on Android YUZU. I have tried Skyline and there is no problem with adding 2nd or 3rd player. I have tried to setup controllers when game is running, no luck.

**Comment by EgoBizarro** [comment]
Oh, Android. Yeah, I don't know if the controller applet is working in that version.
